jetbrains-core/tst/software/aws/toolkits/jetbrains/services/cloudwatch/logs/TableUtilsTest.kt

```diff
@@ -13,33 +12,28 @@ import org.mockito.kotlin.mock
 import org.mockito.kotlin.whenever
 import software.amazon.awssdk.services.cloudwatchlogs.model.LogStream
 import software.aws.toolkits.jetbrains.services.cloudwatch.logs.editor.LogStreamsDateColumn
-import software.aws.toolkits.jetbrains.services.cloudwatch.logs.editor.TimeFormatConversion
+import software.aws.toolkits.jetbrains.services.cloudwatch.logs.editor.LogStreamsStreamColumn
 import java.text.SimpleDateFormat
-import java.time.LocalDate
-import java.time.LocalTime
-import java.time.ZoneOffset
+import java.time.ZoneId
+import java.time.ZonedDateTime
 import java.time.format.DateTimeFormatter
 import java.util.Locale
-import javax.swing.RowSorter
 import javax.swing.SortOrder
 
 class TableUtilsTest {
     @Test
-    fun `test short MDY sorting`() {
-        val dateFormat = SimpleDateFormat("M/d/yy", Locale.ENGLISH)
+    fun `sorting must be enabled to work`() {
+        assertThat(LogStreamsDateColumn(sortable = false).comparator).isNull()
```

Comment:
```suggestion
        assertThat(LogStreamsDateColumn(sortable = false).comparator).isNull
```
?

Reply:
`isNull` does not work, and I am not really sure why

Reply:
```
  /** {@inheritDoc} */
  @Override
  public void isNull() {
    objects.assertNull(info, actual);
  }

  /** {@inheritDoc} */
  @Override
  public SELF isNotNull() {
    objects.assertNotNull(info, actual);
    return myself;
  }
```

Ah, its because isNull() is void